config.js

```diff
@@ -18,20 +18,17 @@ module.exports = {
 	'fenced-code-marker': '`',
 	'file-extension': true,
 	'final-newline': true,
-	// TODO: because of https://github.com/wooorm/remark-lint/issues/77
```

Comment:
Do not remove the `TODO` comments. Those are just warnings but are here for a reason.